src/App.tsx

```diff
@@ -17,18 +63,49 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                setFilterBy={setFilterBy}
+                filterBy={filterBy}
+                setQuery={(queryFromInput) => {
+                  setQuery(queryFromInput);
+                }}
```

Comment:
You could rewrite the same way in simular code all over the task.
```suggestion
                setQuery={setQuery}
```

Reply:
This comment still actual.